app/models/katello/content_view_erratum_filter_rule.rb

```diff
@@ -13,8 +17,20 @@ class ContentViewErratumFilterRule < Katello::Model
     validates :errata_id, :uniqueness => { :scope => :content_view_filter_id }, :allow_blank => true
     validates_with Validators::ContentViewErratumFilterRuleValidator
 
+    validates :date_type,
+      :unless => proc { |o| o.start_date.blank? || o.end_date.blank? },
+      :inclusion => {
+        :in => DATE_TYPES,
+        :allow_blank => false,
+        :message => (_("must be one of the following: %s") % DATE_TYPES.join(', '))
+      }
+
     def filter_has_date_or_type_rule?
       filter.erratum_rules.any? { |rule| rule.start_date || rule.end_date || !rule.types.blank? }
     end
+
+    def pulp_date_type
```

Comment:
They just *happen* to match exactly the same string as in pulp but to a developer this would maybe not be obvious and could easily be broken.  So i asked for a 'translation' to make that more explicit.